frame/vesting/src/lib.rs

```diff
@@ -179,29 +179,6 @@ pub mod pallet {
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
-		#[cfg(feature = "try-runtime")]
```

Comment:
@emostov please confirm this is not used.

Reply:
I can confirm this is no longer necessary for Kusama or Polkadot